lib/zendesk_apps_tools/translate.rb

```diff
@@ -95,6 +95,14 @@ def self.source_root
     end
 
     no_commands do
+      def fetch_locale_async(locale, app_package)
+        Thread.new do
+          say("Fetching #{locale['locale']}")
```

Comment:
Nice work guys!
On Wed, 20 Apr 2016 at 6:13 PM, Chris Atkins <notifications@github.com>
wrote:

> In lib/zendesk_apps_tools/translate.rb
> <https://github.com/zendesk/zendesk_apps_tools/pull/119#discussion_r60366441>
> :
>
> > @@ -95,6 +95,14 @@ def self.source_root
> >      end
> >
> >      no_commands do
> > +      def fetch_locale_async(locale, app_package)
> > +        Thread.new do
> > +          say("Fetching #{locale['locale']}")
>
> If the output gets minced by the multithreading, you might have to set $stdout
> = true at the start of the command.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/zendesk_apps_tools/pull/119/files/ba9072876e57675b5196b681ae136b0cf910c4f7#r60366441>
>